.github/workflows/ci.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Perform build of listenbrainz-rs
+    - uses: actions/checkout@v4
+    - name: Build
```

Comment:
To maintain consistency, this should be called 'Run tests'.

OTOH, `cargo test` will build the library regardless, so should we just forgo the build and go straight to the test?

Reply:
Agree on the first point :+1:

A separate build step makes it easier to see if there was a compile error as opposed to a failing test. `cargo test` should just reuse the built artifacts (afaik) so no problems there.

Reply:
Good point. I suppose it's not much of a blocker. I will approve now...